ArcGISRuntimeSDKQt_CppSamples/UtilityNetwork/CreateLoadReport/README.metadata.json

```diff
@@ -0,0 +1,58 @@
+{
+    "category": "UtilityNetwork",
```

Comment:
I just noticed that our category on the developer site is `UtilityNetwork` - https://developers.arcgis.com/qt/qml/sample-code/

Probably would be preferable to have this sentence case like the other categories. This can be tackled as a separate issue